pkgs/development/libraries/libebur128/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, cmake, speexdsp, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "libebur128-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/jiixyj/libebur128.git";
+    rev = "refs/tags/v1.0.2";
```

Comment:
You can use string interpolation to insert the version number, i.e. `rev = "refs/tags/${version}";`